modules/integration_aws-direct-connect/conf/00-heartbeat.yaml

```diff
@@ -0,0 +1,13 @@
+## Example
+module: aws-direct-connect
```

Comment:
Use the same module name as the other files (and remove ## Example line)

Reply:
Thanks for the feedback, it corrected now. Can you check it plz ?